Makefile

```diff
@@ -133,7 +133,7 @@ test-python-ci: create-db
 		tests/sentry \
 		tests/sentry_plugins \
 		tests/symbolicator \
-		--cov . --cov-report="xml:.artifacts/python.coverage.xml"
+		--cov . --cov-report="xml:.artifacts/python.coverage.xml" --cov-context=test
```

Comment:
This probably needs a comment if you don't want it to get removed.

I usually aggressively delete code I don't know the purpose of (then re-add with comment when/if I find out it was necessary).

Reply:
The comment is above the job run... Idk if you can have comments in the middle of the command, that seemed safer :E

Reply:
cov-context=test has performance implications -- please make sure this doesn't regress test timings significantly

Reply:
There isn't much that I can do here.
Using this option is something that is absolutely necessary to ATS at this time.

You can check the `backend` tests runs for this PR (https://github.com/getsentry/sentry/actions/runs/6457587999/job/17529472864) , which include the flag, and compare it against runs for other PRs that don't include it.

I'll paste a few here for comparison. Notice that I'll sort these. Because there's randomization on the test orders I don't think comparing the job number is actually fair.

| `--cov-context` | commit | backend tests time ordered (ASC) |
|--------|--------|--------|
| ✅ | 3b8238dad6c497a8e3f1f81f3b8a52237d607717 | 20m36 21m34 21m54 22m08 22m14 24m22 28m47 31m13 33m57 35m06 37m47 |
| ✅ | dab7bfae1b0daa5d2be1fe1ee9960eb06b54c9fc | 21m17 24m37 25m45 26m34 27m20 28m28 29m14 30m42 31m01 31m37 34m32 |
| 🚫 | bb17241648050f2a2b749a963ef1eec6f30a390d | 16m36 18m16 20m08 20m18 21m18 21m55 22m 08 24m22 24m36 24m51 32m19 |
| 🚫 | d36da19cf57433408068d100cab5b1e56ff6b8de | 17m38 18m04 19m28 19m43 20m10 23m16 23m25 25m33 27m10 28m31 35m55 |